pkg/controller/secret-mirror_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/openshift/ci-secret-mirroring-controller/pkg/controller/config"
```

Comment:
We usually place local repo imports last

Reply:
Done.